mrp_business/models/mrp_bom.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (C) 2022 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+
+
+class MrpBom(models.Model):
+    _inherit = "mrp.bom"
+
+    # ========== Divers
+    currency_id = fields.Many2one(related="product_tmpl_id.currency_id")
+    description_packaging = fields.Char(string="Packaging description")
+    description_short = fields.Char(string="Short description")
+    description_long = fields.Text(string="Long description")
+    # Tracking not possible for One2many
+    # bom_line_ids = fields.One2many(track_visibility="onchange")
+    grade = fields.Selection(
```

Comment:
```suggestion
    priority = fields.Selection(
```

(nom standard dans odoo, pour ce concept). (utilisé dans project par exemple).

Reply:
ok !